docs/api/api/api.txt

```diff
@@ -2012,7 +2012,9 @@ GET /staging/<staging_workflow_project>/staging_projects/<staging_project>/stage
 XmlResult: staged_requests
 
 POST /staging/<staging_workflow_project>/staging_projects/<staging_project>/staged_requests
-  Add one or multiple bs_requests to the staged requests of a staging project.
+  Add one or multiple bs_requests to the staged requests of a staging project
+  remove_exclusion: If remove_exclusion is set (any value will work) it will allow the removal of the exclusions over a request before staging a request.
```

Comment:
If I read the code right, a value of 0 will not 'work'. the api.txt is generally bad in giving valid values - so in line with other paragraphs in the same document I find: `remove_exclusion: removes the request from exclusion before staging` to be enough.

Reply:
If you put a '0' in it, it would behave as if you put a '1' in it. That's what I wanted to say with `any value will work`. But let's use your suggestion.

Reply:
ah, I always forget that ruby is not perl :)

Reply:
```
~>perl -e 'my $a="0"; print "no\n" unless $a;'
no
```